src/main/resources/org/jenkinsci/plugins/workflow/steps/WaitForConditionStep/config.jelly

```diff
@@ -24,4 +24,8 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core"/>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="initialRecurrencePeriod" title="Initial Recurrence Period Milliseconds">
```

Comment:
Is it worth adding a help page (`help-intialRecurrencePeriod.html`) for this field?